critiquebrainz/frontend/templates/mapping/spotify.html

```diff
@@ -47,18 +47,7 @@ <h2>{{ _('Spotify album mapping') }}</h2>
     {% endfor %}
   </div>
   {% if count > limit %}
```

Comment:
Let's move these `count > limit` (this and the ones inside other files modified in this PR) checks to inside the macro.